change-notes/1.20/analysis-java.md

```diff
@@ -15,7 +12,7 @@
 | **Query**                  | **Expected impact**    | **Change**                                                       |
 |----------------------------|------------------------|------------------------------------------------------------------|
 | Arbitrary file write during archive extraction ("Zip Slip") (`java/zipslip`) | Fewer false positive results | Results involving a sanitization step that converts a destination `Path` to a `File` are no longer reported. |
-| Double-checked locking is not thread-safe (`java/unsafe-double-checked-locking`) | Fewer false positive results and more true positive results | Results that use safe publication through a `final` field are no longer reported. Results that initialize immutable types like `String` incorrectly are now reported. |
+| Double-checked locking is not thread-safe (`java/unsafe-double-checked-locking`) | Fewer false positive and more true positive results | Results that use safe publication through a `final` field are no longer reported. Results that initialize immutable types like `String` incorrectly are now reported. |
```

Comment:
It looks like this line can be removed because this query is introduced in this release, so this line should not have been added in the first place.

Reply:
Thanks. I should have spotted that.